io.openems.edge.core/src/io/openems/edge/core/meta/Config.java

```diff
@@ -1,12 +1,18 @@
 package io.openems.edge.core.meta;
 
+import org.osgi.service.metatype.annotations.AttributeDefinition;
 import org.osgi.service.metatype.annotations.ObjectClassDefinition;
 
+import io.openems.edge.common.currency.CurrencyConfig;
+
 @ObjectClassDefinition(//
 		name = "Core Meta", //
 		description = "The global manager for Metadata.")
 @interface Config {
 
 	String webconsole_configurationFactory_nameHint() default "Core Meta";
 
+	@AttributeDefinition(name = "Currency", description = "Currency to be used for energy purchase; Energy price value is converted to appropraite currency based on current exchange rate")
```

Comment:
"appropraite" -> "appropriate"

But I'd just write "Any monetary value is per definition in this Currency. Values that are retreived in different currency (e.g. energy price from a webservice) are converted to this Currency internally based on current exchange rate."

Reply:
Corrected using "Every monetary value is inherently expressed in this Currency. Values obtained in a different currency (e.g. energy prices from a web service) are internally converted to this Currency using the current exchange rate."